firebase.json

```diff
@@ -5,7 +5,12 @@
     "trailingSlash": false,
     "headers": [{
         "source": "**/*.@(txt|json)",
-        "headers": [ { "key": "Access-Control-Allow-Origin", "value": "*" } ]
+        "headers": [
+          { "key": "Access-Control-Allow-Origin", "value": "*" },
+          { "key": "Referrer-Policy", "value":  "strict-origin-when-cross-origin"},
+          { "key": "X-Frame-Options", "value":  "SAMEORIGIN"},
+          { "key": "X-Content-Type-Options", "value":  "nosniff"}
+        ]
```

Comment:
Why only apply this to `txt` and `json` files?

That is what this does, right?

Reply:
I see no reason not to enforce:
```js
          { "key": "Referrer-Policy", "value":  "strict-origin-when-cross-origin"},
          { "key": "X-Frame-Options", "value":  "SAMEORIGIN"},
          { "key": "X-Content-Type-Options", "value":  "nosniff"}
```

on all requests..

If viable, we might also want to consider CSP policies along the lines of:
```
Content-Security-Policy: default-src https:; script-src 'nonce-{random}' 'unsafe-inline'; object-src 'none'; base-uri 'none'
```

Though, we might need to use hash instead of nonce... or allow list specific domains... since this is a static site.

Reply:
You're correct, I didn't mean to limit this to just `txt` and `json` files, I've expanded it to a wildcard.

Reply:
As for the specific domains, I tried going the route of listing specific domains. I haven't set a CSP up before, so if you want to take a closer look, that'd be great! Thanks.